policyengine_us/variables/gov/states/ut/tax/income/credits/retirement_credit/ut_retirement_credit.py

```diff
@@ -7,22 +7,6 @@ class ut_retirement_credit(Variable):
     label = "Utah retirement credit"
     unit = USD
     definition_period = YEAR
-    defined_for = StateCode.UT
+    defined_for = "ut_claims_retirement_credit"
 
-    def formula(tax_unit, period, parameters):
-        claims_retirement_credit = tax_unit(
-            "ut_claims_retirement_credit", period
-        )
-        max_credit = tax_unit("ut_retirement_credit_max", period)
-        limiting_liability = (
-            tax_unit("ut_income_tax_before_credits", period)
-            - tax_unit("ut_taxpayer_credit", period)
-            - tax_unit("ut_eitc", period)
-        )
-        if not parameters(
-            period
-        ).gov.states.ut.tax.income.credits.retirement.refundable:
-            return min_(
-                claims_retirement_credit * max_credit, limiting_liability
-            )
-        return claims_retirement_credit * max_credit
+    adds = ["ut_retirement_credit_max"]
```

Comment:
@nikhilwoodruff you did utah previous right? could you remind me why we did the non-refundability this way initially?

Reply:
👋

Reply:
This was the general pattern for all nonrefundable credits at the time right?